src/intents/Declaration.ts

```diff
@@ -90,6 +91,10 @@ class Declaration {
     return bits
   }
 
+  nonces ():{ bit: Bit, nonce: BigInt }[] {
```

Comment:
should probably also add a nonce() method to the Intent.ts class. maybe this could use it?